src/workloadaccounts.ts

```diff
@@ -38,31 +42,34 @@ export const WorkloadAccounts = (
       organisationUnit
     )
 
-    const conflictingAccount = accounts.workloadAccounts?.find(
-      account => account.email === workloadEmail
-    )
+    const conflictingAccount = workloadAccounts.items.find(
+      account => account.get('email') === workloadEmail
```

Comment:
Unhappy with magic word - but can't think of a clean away around this, maybe we export a value from that specific file like a static.

Reply:
Resolving as out of scope for now, can come back to this in the future if we deem it necessary.